src/librustdoc/clean/mod.rs

```diff
@@ -2735,10 +2735,6 @@ fn clean_maybe_renamed_item<'tcx>(
                 Box::new(clean_ty(ty, cx)),
                 Constant { kind: ConstantKind::Local { body: body_id, def_id } },
             ),
-            ItemKind::OpaqueTy(ref ty) => OpaqueTyItem(OpaqueTy {
```

Comment:
Seems like we need to explicitly handle this and return an empty vec instead.

Reply:
A bit strange that our tests didn't catch this. Do you mind seeing if you can add a test that would have discovered this ICE?